tests/ui/asm/aarch64/type-f16.rs

```diff
@@ -0,0 +1,21 @@
+//@ only-aarch64
+//@ run-pass
+
+#![feature(f16, f128)]
+use std::arch::asm;
+#[inline(never)]
+pub fn f32_to_f16_asm(a: f32) -> f16 {
+    let ret: f16;
+    unsafe {
+        asm!(
+        "fcvt    {ret:h}, {a:s}",
+        a = in(vreg) a,
+        ret = lateout(vreg) ret,
+        options(nomem, nostack),
+        );
+    }
+    ret
+}
+fn main() {
+    f32_to_f16_asm(1.0 as f32);
```

Comment:
```suggestion
    assert_eq!(f32_to_f16_asm(1.0 as f32), 1.0);
```

Since this is now run-pass, might as well verify the output

Reply:
thanks.